### PR TITLE
V0.2 VM scheduling docs

### PR DESCRIPTION
**Pull request by kyu-kuanwei**
# Description

Add VM scheduling documents.

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [ ] New feature
- [ ] Test
- [x] Doc update
- [ ] Docker update

## Related Component

- [ ] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [ ] Linux
- Python version:
  - [ ] 3.6
  - [ ] 3.7
- Key information snapshot(s):
  
## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [x] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: notebooks/data_center/VM Data Analytics.ipynb [hunk 1]
```diff
@@ -0,0 +1,1152 @@
+{
```
Comment (Jinyu-W) [inline notebooks/data_center/VM Data Analytics.ipynb:1]: I remember this file already exists with a lower case name. Check it.

File: docs/source/scenarios/vm_scheduling.rst [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+good VM allocation strategy can maximize the resourece utilization and thus can increase the profit by 
+providing more VMs to users. For cloud users, a good VM allocation strategy can 
+minimize the VM response time and have a better using experience. We hope this scenario can meet 
+the real needs and provide you with a demand simulation that is closest to the real situation.
+
+
+Resource Flow
+--------------
+
+In this scenario, the physical resources in each physical machine (PM) are the
```
Comment (Jinyu-W) [inline docs/source/scenarios/vm_scheduling.rst:18]: already mentioned ``(PM)``before

**Review by Jinyu-W — CHANGES_REQUESTED** [review]

File: notebooks/data_center/VM Data Analytics.ipynb (hunk 1 above)
Comment (kyu-kuanwei) [reply notebooks/data_center/VM Data Analytics.ipynb:1]: Yes, I found that there are two folders datacenter and vm_scheduling.

File: docs/source/scenarios/vm_scheduling.rst [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+**azure.2019.10k**\ : We randomly sampled 10,000 VMs from the AzurePublicDatasetV2.
+
+**azure.2019.336k**\ : We randomly sampled 336,000 VMs from the AzurePublicDatasetV2.
+
+Naive Baseline
+^^^^^^^^^^^^^^^
+
+Belows are the final environment metrics of the method random allocation and best-fit allocation in 
+different topologies. For each experiment, we setup the environment and test for a duration of 30 days.
```
Comment (Jinyu-W) [inline docs/source/scenarios/vm_scheduling.rst:126]: the PM setting is listed as a single column, you can mention it and why we do this here.

To test under different resource supply/PM capacity?

**Review by Jinyu-W — CHANGES_REQUESTED** [review]

File: docs/source/scenarios/vm_scheduling.rst [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+and process the data files. Afterwards, if you want to run multiple simulations, the system will detect
+whether the processed data files exist or not. If not, it will then trigger the pipeline again. Otherwise,
+the system will reuse the processed data files. 
+
+
+Environment Interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+Before starting interaction with the environment, we need to know the definition of ``DecisionPayload`` and 
+``Action`` in VM Scheduling scenario first. Besides, you can query the environment snapshot list to get more
```
Comment (Jinyu-W) [inline docs/source/scenarios/vm_scheduling.rst:273]: add link for snapshot list here

Comment (Jinyu-W) [reply docs/source/scenarios/vm_scheduling.rst:273]: https://maro.readthedocs.io/en/latest/key_components/data_model.html#advanced-features

**Review by Jinyu-W — CHANGES_REQUESTED** [review]

**Review by Jinyu-W — APPROVED** [review]

**Pull request merged**